hugolib/page_collections.go

```diff
@@ -43,6 +43,9 @@ type PageCollections struct {
 	// Includes absolute all pages (of all types), including drafts etc.
 	rawAllPages Pages
 
+	// Includes headless bundles, i.e. bundles that produce no output for its content page.
+	headlessPages Pages
```

Comment:
Is there any reason why this should be private?
If we can make it public, it will be possible to generate one-page sites constructed from a bunch of page bundles, without "single" view of any of them. What do you think?

Reply:
I don't understand what you are talking about, which answers your question.

Reply:
The thing, when in doubt, keep it private. I have been bitten by this enough time to have learned.

Reply:
I mean if I want to iterate through all the headless pages in my template, how can I do that?

Reply:
You cannot. And that is out of scope for this feature. If you somehow want to do that, you will have to discuss it in another topic.